frontend/src/plant/AddPlant.tsx

```diff
@@ -0,0 +1,24 @@
+import {FormEvent, useState} from "react";
+import {toast, ToastContainer} from "react-toastify";
+import 'react-toastify/dist/ReactToastify.css';
+
+type AddPlantProps = {
+    addPlant: (name: string) => void;
+}
+export default function AddPlant(props: AddPlantProps) {
+    const [addPlantString, setAddPlantString] = useState<string>('')
+    const submitInput = (event: FormEvent<HTMLFormElement>) => {
+        if (addPlantString !== "") {
+            props.addPlant(addPlantString);
+            setAddPlantString('')
```

Comment:
you could only call `setAddPlantString` when the `addPlant` - axios request was successful